htsohm/simulation/methane_loading.py

```diff
@@ -146,8 +146,12 @@ def run(run_id, material_id, helium_void_fraction):
             print("Simulating methane loading in %s-%s..." % (run_id, material_id))
             subprocess.run(['simulate', './MethaneLoading.input'], check=True, cwd=output_dir)
 
-            filename = "output_%s-%s_1.1.1_298.000000_3.5e+06.data" % (run_id, material_id)
-            output_file = os.path.join(output_dir, 'Output', 'System_0', filename)
+            file_name_part = "output_%s-%s" % (run_id, material_id)
+            output_subdir = os.path.join(output_dir, 'Output', 'System_0')
+            for file in os.listdir(output_subdir):
```

Comment:
never use 'file' as a variable because it is a built-in method (https://docs.python.org/3.6/library/functions.html). It should work in most cases, but it will annoy any other programmer looking at your code b/c it screws up their syntax highlighting... :)